host/xtest/regression_1000.c

```diff
@@ -2036,3 +2037,190 @@ static void xtest_tee_test_1025(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1025, xtest_tee_test_1025,
 		 "Test memref NULL and/or 0 bytes size");
+
+#define TEE_UUID_NS_NAME_SIZE  128
+
+/*
+ * TEE Client UUID name space identifier (UUIDv4)
+ *
+ * Value here is random UUID that is allocated as name space identifier for
+ * forming Client UUID's for TEE environment using UUIDv5 scheme.
+ */
+static const char *client_uuid_linux_ns = "58ac9ca0-2086-4683-a1b8-ec4bc08e01b6";
+
+/* TEEC_LOGIN_PUBLIC's Client UUID is NIL UUID */
+static TEEC_UUID client_uuid_public = { };
+
+static TEEC_Result get_client_identity(TEEC_Session *session, uint32_t *login,
+				       TEEC_UUID *client_uuid)
```

Comment:
If moved there I suppose `ta_os_test_cmd_client_identity()` would be more matching?

There is already `ta_crypt_cmd_*` variants and this would fit to that scheme.

Reply:
Updated